api/src/test/java/gov/cms/ab2d/api/controller/BulkDataAccessAPIIntegrationTests.java

```diff
@@ -98,4 +100,26 @@ public void testPatientExportWithInvalidOutputFormat() throws Exception {
                 .andExpect(jsonPath("$.issue[0].code", Is.is("invalid")))
                 .andExpect(jsonPath("$.issue[0].details.text", Is.is("IllegalArgumentException: An _outputFormat of Invalid is not valid")));
     }
+
+    @Test
+    public void testDeleteJob() throws Exception {
+        this.mockMvc.perform(get(API_PREFIX + PATIENT_EXPORT_PATH).contentType(MediaType.APPLICATION_JSON));
+        Job job = jobRepository.findAll(Sort.by(Sort.Direction.DESC, "id")).iterator().next();
+
+        System.out.println(job.getJobID());
+
+        this.mockMvc.perform(delete(API_PREFIX + "/Job/" + job.getJobID() + "/$status"))
+            .andExpect(status().is(202))
+            .andExpect(content().string(JOB_CANCELLED_MSG));
+
+        Job cancelledJob = jobRepository.findByJobID(job.getJobID());
+        Assert.assertEquals(JobStatus.CANCELLED, cancelledJob.getStatus());
+    }
+
+    @Test
+    public void testDeleteNonExistentJob() throws Exception {
+        this.mockMvc.perform(delete(API_PREFIX + "/Job/NonExistentJob/$status"))
+                .andExpect(status().is(404))
+                .andExpect(content().string(JOB_NOT_FOUND_ERROR_MSG));
+    }
```

Comment:
Will need test cases to cover attempts to cancel FAILED or COMPLETED job (see previous comments)